packages/PhpDocParser/PhpDocParser/PhpDocNodeVisitor/CloningPhpDocNodeVisitor.php

```diff
@@ -18,7 +18,11 @@ final class CloningPhpDocNodeVisitor extends AbstractPhpDocNodeVisitor
     public function enterNode(Node $node): Node
     {
         $clonedNode = clone $node;
-        $clonedNode->setAttribute(PhpDocAttributeKey::ORIG_NODE, $node);
+
+        if (! $clonedNode->hasAttribute(PhpDocAttributeKey::ORIG_NODE)) {
+            $clonedNode->setAttribute(PhpDocAttributeKey::ORIG_NODE, $node);
+        }
```

Comment:
this is essential to ensure when orig node exists, no need to re-fill with node, use whatever ORIG_NODE value is, eg: `null` for reprint.